utils/utils.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"strconv"
+
+	stakeibctypes "github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+func FilterDepositRecords(arr []stakeibctypes.DepositRecord, condition func(stakeibctypes.DepositRecord) bool) (ret []stakeibctypes.DepositRecord) {
+	for _, elem := range arr {
+		if condition(elem) {
+			ret = append(ret, elem)
+		}
+	}
+	return ret
+}
+
+func Int64ToCoinString(amount int64, denom string) string {
```

Comment:
Haha great idea, I've been recreating int64=>string, string=>int64, int64=>coinstring all over the place. Will def use this and add to this file with a few other helpers

Reply:
What error handling are you thinking we should add?